### PR TITLE
AP-5504 Update README.md

### PR DESCRIPTION
**Pull request by mrspillingsley**
Daddddd I need your approval @chriskemptondoc

### CODE REVIEW

File: v2/README.md [hunk 1]
```diff
@@ -15,4 +15,37 @@
...
+   
+      f. Unlawful Activities: Using the API for illegal or unethical activities, including activities that violate privacy rights or applicable laws.
+
+4. #### Compliance and Monitoring:
+   DocNetwork reserves the right to monitor the User's use of the API to ensure compliance with this Policy. Users may be required to provide documentation demonstrating their compliance with data protection laws and regulations.
+
+6. #### Consequences of Non-Compliance:
+   Violation of this Policy may result in the suspension or termination of API access, as well as legal actions if deemed necessary. DocNetwork reserves the right to take appropriate measures to address non-compliance.
+
+8. ####Changes to Policy:
```
Comment (chriskemptondoc) [inline v2/README.md:47]: Looks like one too many #s here.

**Review by chriskemptondoc — CHANGES_REQUESTED** [review]
Markdown oopsie

File: v2/README.md (hunk 1 above)
Comment (mrspillingsley) [reply v2/README.md:47]: we don't want to use Header level 4? Fineeee I'll change it

Comment (mrspillingsley) [reply v2/README.md]: It was missing a space between the # and the word - updated and matches the theme for rest of text

**Review by chriskemptondoc — APPROVED** [review]
Looks great!  I'll merge.

**Pull request merged**